### PR TITLE
Subm 2.7 updates stolostron/backlog#26797

### PR DESCRIPTION
**Pull request by oafischer**
https://github.com/stolostron/backlog/issues/26797

### CODE REVIEW

File: release_notes/known_issues.adoc [hunk 1]
```diff
@@ -1060,39 +1060,16 @@ Managed clusters that are manually imported on the primary hub cluster show a `P
...
 
-You can use headless services with Globalnet, except when you access the exported headless service from a client that resides in the same cluster by using the `clusterset.local` domain name. When you use the `clusterset.local` domain name to access the headless service, the `globalIP` that is associated with the headless service is not routable in the cluster and is returned to the client. 
-
-You can use the `cluster.local` domain name to access the local headless services.
-
-[#submariner-airgap]
-=== Air-gapped clusters are not supported
-
-Submariner is not validated for clusters that are provisioned in an air-gapped environment.
+Headless services without selectors is only supported in Globalnet. Headless services with selectors is supported in both Globalnet and Lighthouse.
```
Comment (nyechiel) [inline release_notes/known_issues.adoc]: @sridhargaddam can you come up with a better text that describes what does it mean for a user? this text is accurate from a dev perspective... like are we saying that connectivity is supported but not DNS?

Comment (sridhargaddam) [reply release_notes/known_issues.adoc]: > like are we saying that connectivity is supported but not DNS?

Yes

What are your thoughts on the following sentence?

Service discovery is not supported for Headless services without selectors when using Globalnet.

Comment (nyechiel) [reply release_notes/known_issues.adoc]: Much better.

But looking at the [upstream notes](https://submariner.io/community/releases/#v0130) we do mention DNS: "Added support for connecting to Kubernetes headless Services without Pod label selectors in Globalnet deployments. This is useful when you want to point a Service to another Service in a different namespace or external network. When endpoints are manually defined by the user, Submariner automatically routes the traffic and provides DNS resolution."

Comment (sridhargaddam) [reply release_notes/known_issues.adoc]: Thanks for digging and updating Nir. I think "...**provides DNS resolution**" is probably a typo as the corresponding [PR in lighthouse](https://github.com/submariner-io/lighthouse/pull/818) is still under review. Anyways, lets hear from @vthapar @aswinsuryan.

Comment (sridhargaddam) [reply release_notes/known_issues.adoc]: Based on the slack conversation, @aswinsuryan confirmed that Service Discovery is not supported even for Vanilla SM.

So, I feel we can use the following note

Service discovery is not supported for Headless services without selectors.

File: release_notes/known_issues.adoc [hunk 2]
```diff
@@ -1060,39 +1060,16 @@ Managed clusters that are manually imported on the primary hub cluster show a `P
...
 
-You can use headless services with Globalnet, except when you access the exported headless service from a client that resides in the same cluster by using the `clusterset.local` domain name. When you use the `clusterset.local` domain name to access the headless service, the `globalIP` that is associated with the headless service is not routable in the cluster and is returned to the client. 
-
-You can use the `cluster.local` domain name to access the local headless services.
-
-[#submariner-airgap]
-=== Air-gapped clusters are not supported
-
-Submariner is not validated for clusters that are provisioned in an air-gapped environment.
+Service discovery is not supported for headless services without selectors.
```
Comment (sridhargaddam) [inline release_notes/known_issues.adoc]: @vthapar mentioned on slack that non-globalnet deployments it should work. See here - https://kubernetes.slack.com/archives/C010RJV694M/p1669917356964549?thread_ts=1669882931.654769&cid=C010RJV694M

@oafischer can you please change the description as follows.


```suggestion
Service discovery is not supported for headless services without selectors when using Globalnet.
```

**Comment by oafischer** [comment]
@nyechiel @sridhargaddam Can we merge this?

File: release_notes/known_issues.adoc (hunk 2 above)
Comment (sridhargaddam) [reply release_notes/known_issues.adoc]: @oafischer Looks like this was accidentally marked as resolved, can you please include the requested change.

**Comment by sridhargaddam** [comment]
> @nyechiel @sridhargaddam Can we merge this?

Please take a look at my last comment.

File: release_notes/known_issues.adoc [hunk 3]
```diff
@@ -986,34 +986,16 @@ Managed clusters that are manually imported on the primary hub cluster show a `P
...
-
 [#headless-services-globalnet]
-=== Headless services with Globalnet is not supported in some cases
+=== Limited headless services support
 //2.5:24159
 
-You can use headless services with Globalnet, except when you access the exported headless service from a client that resides in the same cluster by using the `clusterset.local` domain name. When you use the `clusterset.local` domain name to access the headless service, the `globalIP` that is associated with the headless service is not routable in the cluster and is returned to the client. 
-
-You can use the `cluster.local` domain name to access the local headless services.
+Service discovery is not supported for headless services without selectors.
```
Comment (sridhargaddam) [inline release_notes/known_issues.adoc]: @oafischer Looks like this was accidentally marked as resolved, can you please include the following change and we are good to go.

Service discovery is not supported for headless services without selectors when using Globalnet.

Comment (oafischer) [reply release_notes/known_issues.adoc]: Done!

**Comment by sridhargaddam** [comment]
Thanks @oafischer 

/lgtm

**Pull request merged**